tests/libcxx/expected_results.txt

```diff
@@ -31,6 +31,12 @@ std/language.support/support.exception/propagation/rethrow_exception.pass.cpp FA
 # Testing nonstandard behavior
 std/utilities/template.bitset/bitset.cons/string_ctor.pass.cpp FAIL
 
+# libc++ doesn't implement LWG-3648: "format should not print bool with 'c'"

```

Comment:
It's weird to add new skips in the middle of a section; could we please move this down to line 88-ish? (And in `skipped_tests.txt` as well).

Reply:
(This is simple and mechanical, I'll just apply the change.)